README.md

```diff
@@ -88,6 +88,10 @@ Elasticsearch is used when `STORAGE_TYPE=elasticsearch`. The schema is compatibl
                   recommended to set this to all the master nodes of the cluster.
     * `ES_NODES_WAN_ONLY`: Set to true to only use the values set in ES_HOSTS, for example if your
                            elasticsearch cluster is in Docker. Defaults to false
+    * `ES_USERNAME` and `ES_PASSWORD`: Elasticsearch basic authentication. Use when X-Pack security
+                                       (formerly Shield) is in place. By default no username or
+                                       password is provided to elasticsearch.
+    * `ES_SSL`: Set to `true` when connections to elasticsearch must use TLS/SSL. Defaults to `false`.
```

Comment:
What I'd do here is emulate upstream. So we allow https:// format for the ES_HOSTS https://github.com/openzipkin/zipkin/blob/master/zipkin-server/README.md#elasticsearch-storage (eventhough upstream currently only allows for a single https target, we can do more than one here)

So the idea is to parse ES_HOSTS and if one of the hosts has https in it, flip the TLS flag.